pkg/utils/log.go

```diff
@@ -138,6 +138,15 @@ func (l *Logger) Warning(format string, v ...interface{}) {
 	}
 }
 
+// Log error variable as a warning
+func (l *Logger) Warn(err error) error {
```

Comment:
But you may want to do this in another PR.  I do not think you need this in this PR.